src/util/expr_util.cpp

```diff
@@ -289,3 +296,10 @@ exprt make_binary(const exprt &expr)
 
   return previous;
 }
+
+bool expr_has_float(const exprt &src)
```

Comment:
Wouldn't it be more consistent with the rest of the API to call this `expr_has_floatbv`?

Reply:
Renamed it

Reply:
Thanks!